CONTRIBUTING.md

```diff
@@ -82,26 +80,8 @@ The secure way to configure your monitoring.               <\
 
 ```
 
-The API is now available :).
+The API backend is now available on port 8080 :).
 
 ### Web App
 
-To be able to use the Web app, you will have to follow the steps to run the API first. For the moment there is no
-particular desire to run the Web-app on the development environment without the API at the same time.
-
-* Install nodejs [version 14 or greater](https://nodejs.org/).
-* Install npm [version 7 or greater](https://www.npmjs.com/).
-* Then go to the folder `./ui`, and install all necessary packages:
-
-```bash
-cd ./ui
-npm install
-```
-
-* Finally, start the web-app:
-
-```bash
-npm start
-```
-
-The web-app is now available using the url http://localhost:3000.
+See the [ui/README.md](./ui/README.md) file for details around the build process and the structure of the web UI. The web UI currently still shows a canned test dashboard that is not loaded from the Perses API backend yet, but it already runs health checks that require the backend to be running in order to not show errors in the UI.
```

Comment:
perhaps we should keep the minimum version required to run the frontend don't you think ?
Here or in the `ui/readme` ?

Reply:
I'm wondering if we should put nodejs 16 instead. Not sure if it really works with nodejs 14.
Since we are running nodejs 16 in the CI, probably better to aligned the version with it.

Reply:
Added that to `ui/README.md` and specified Node 16 there.